README.md

```diff
@@ -1,15 +1,15 @@
 # JSpecify
 
-**This project is still under active development. Any and all aspects of
-the API and specifications are subject to change prior to the 1.0 release.
-Using the annotations in a released library is not recommended.**
-
 An artifact of well-specified annotations to power static analysis checks and
 JVM language interop.  Developed by consensus of the partner organizations
 listed at our main web site, [jspecify.org](http://jspecify.org).
 
 Our current focus is on annotations for nullness analysis.
 
-## Stuff to read
+## Status
+
+[Version 0.3](https://github.com/jspecify/jspecify/releases/tag/v0.3.0) is relatively safe to depend on in your code. Or you can read [a more detailed answer](https://github.com/jspecify/jspecify/wiki/adoption).
```

Comment:
"For caveats, see \[this more detailed answer\]?"

"Or" sounds just slightly odd as a way to start the sentence. But if nothing better jumps out to you, you're welcome to merge as-is.

Reply:
I get it but will take the easy path for now.